wtf/wtf.h

```diff
@@ -4,7 +4,8 @@
 
 #define sucks ==nullptr
 #define does_not_suck !=nullptr
-#define contains_shit empty()
+#define contains_shit(X) !(X).empty()
```

Comment:
Alternately would a redefinition of containsShit as .size() > 0 work to keep it in the same style as doesnt_contain_shit?
that is 
```
x.contains_shit;
x.doesnt_contain_shit;
```
as opposed to 
```
contains_shit(x);
x.doesnt_contain_shit;
```

Reply:
+1

Reply:
@TNychka, I like the idea, feel free to create another PR for that :)